src/Catalog/Registration/RegistrationCollector.cs

```diff
@@ -66,6 +66,8 @@ protected override Task<IEnumerable<CatalogItemBatch>> CreateBatches(IEnumerable
             // from the correct location (even though we may have
             // a little rework).
 
+
+            // TODO: this is where processing of a package starts...
```

Comment:
If we want to measure the processing duration of a single package in Catalog2Registration, I recommend this point to start the measuring